meson_options.txt

```diff
@@ -1,5 +1,6 @@
 option('build', type : 'combo', choices : ['all', 'standalone', 'library'], value : 'all', description : 'build type')
 option('agent', type : 'boolean', value : true, description : 'enable the fwupd agent')
+option('bsd', type : 'boolean', value : false, description : 'disable linux specific options')
```

Comment:
I don't think we want an option for this - it should all be detectable at build time.

Reply:
Agree.